digits/dataset/images/classification/test_views.py

```diff
@@ -382,7 +429,6 @@ def check_min_per_class(self, type):
         assert self.delete_dataset(job_id) == 200, 'delete failed'
         assert not self.dataset_exists(job_id), 'dataset exists after delete'
 
-
```

Comment:
I never thought of blank line removal as an issue.  Could cause merge problems I suppose.